Jojibrush.pro

```diff
@@ -52,7 +52,8 @@ HEADERS += \
         shapes/Shapes.hpp \
         shapes/Polygon.hpp \
         shapes/Shape.hpp \
-        shapes/Text.hpp
+        shapes/Text.hpp \
+    drawingutility.hpp
```

Comment:
God Qt annoys the fuck out of me for this reason, plz fix

Reply:
I'm pretty sure 1 indentation is the default, and we just keep adding another because we already started doing that LOL

Reply:
it looks pretty tho :(